lib/source-destination/file.ts

```diff
@@ -181,6 +181,7 @@ export class File extends SourceDestination {
 		highWaterMark,
 	}: { highWaterMark?: number } = {}): Promise<NodeJS.WritableStream> {
 		// TODO: use SourceDestinationFs (implement write) when node 14 becomes LTS
+		// eslint-disable-next-line
```

Comment:
Instead of disabling the rule, we should just switch the `@ts-ignore` to `@ts-expect-error`.

Reply:
there's one already, this disable-line is to about the ban-ts-comment.
I've turned off ban-ts-comment globally otherwise i need to go down a rabbit hole that is out of scope